core/src/test/java/hudson/BulkChangeTest.java

```diff
@@ -1,18 +1,18 @@
 /*
  * The MIT License
- * 
+ *
```

Comment:
Please avoid gratuitous (for example, whitespace-only) changes in diffs in PRs. They make the diff much harder to read, increase the change of merge conflicts, and cause lots of trouble down the road when cherry-picking, running `git-blame`, etc.